images/baremetal/Dockerfile.ci

```diff
@@ -2,6 +2,7 @@
 # It builds an image containing only the openshift-install.
 
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+RUN yum install -y libvirt-devel
```

Comment:
Current pattern is to chain this up to a trialing `yum clean all`, like [this][1].

[1]: https://github.com/openshift/installer/blob/8f972b45987a32cc91bc61c39a727e9a1224693d/images/installer/Dockerfile.upi.ci#L19-L25

Reply:
Also, won't you want all of [these][1]?  If not, why not?

[1]: https://github.com/openshift/installer/tree/8f972b45987a32cc91bc61c39a727e9a1224693d/docs/dev/libvirt#install-and-enable-libvirt

Reply:
I don't think anything other than libvirt-devel is needed to compile, we're not actually running the installer or really interacting with libvirt, so the rest isn't needed (I think).

Reply:
That makes sense, but makes me wonder why the baremetal installer needs to set the `libvirt` build tag at all?

Reply:
We use a libvirt provisioning host to bootstrap the cluster

Reply:
To be clear, those full set of packages are needed wherever you would run the installer from, but they’re not needed to build it

Reply:
Can we include them here then?  That way folks who wanted to could run the baremetal installer via a container runtime, vs. extracting just the binary and having it dynamically link to local libraries.

Reply:
That'd be a little awkward since it'd be so different from how the installer runs elsewhere.  It's not a big deal to depend on libvirt.  We're only supporting RHEL 8 hosts.

Reply:
Also I should mention this is just the build container, we only need devel here. The actual container we're distributing (below this section) has libvirt-libs, so the installer could run inside the container runtime, and talk to an *external* libvirt.  

Having libvirt itself run inside the container would be complicated, we'd need to do many of the things kubervirt does to make that happen.